Sources/Build/BuildPlan.swift

```diff
@@ -187,6 +187,8 @@ public final class ClangTargetDescription {
         // Only enable ARC on macOS.
       #if os(macOS)
         args += ["-fobjc-arc"]
+      #elseif os(Linux)
+        args += ["-fblocks"]
```

Comment:
This should probably just be added unconditionally, like `-fmodules`. It doesn't do any harm on Apple platforms, and it'll be necessary on BSD and Systemz and all the others.

Reply:
Agreed

Reply:
also when cross-compiling to Linux, you definitely want to have `-fblocks` set.